### DISCUSSION TITLE
Install FreshRSS without the need for a sepete server PC.

### DISCUSSION THREAD

**Discussion by whatisyourname69** [FreshRSS/FreshRSS#5761] [Q&A] [answered]
Hi, 

I don't have a separate PC on hand to install a server version of Linux. How can  Install FreshRSS directly on Linux Mint?

**Comment by Frenzie** [comment]
There are basically two ways.

1. Install Apache, PHP and other dependencies
https://freshrss.github.io/FreshRSS/en/admins/02_Prerequisites.html

2. Use Docker

If you search for those terms in combination with Ubuntu I imagine something usable on Mint should exist, but I don't know any tutorials otoh I'm afraid.

**Comment by Alkarex** [comment, marked as answer]
I would also recommend running FreshRSS through our ready-to-use Docker image, except if you are familiar running a Web server locally with its configuration.

Here is a 2-step suggestion to try:

```console
sudo apt install docker-compose

sudo docker run -d --restart unless-stopped --log-opt max-size=10m -p 8080:80 -e TZ=Europe/Paris -e 'CRON_MIN=5,35' -v freshrss-data:/var/www/FreshRSS/data --name freshrss freshrss/freshrss
```

And then you can access FreshRSS in your Web browser from http://localhost:8080/

See also https://github.com/FreshRSS/FreshRSS/blob/edge/Docker/README.md#quick-run

Just ask again if the commands above do not work.

**Reply by whatisyourname69** [reply]
This has worked but I have little to now idea how. I mean is been the quickest deployment of freshrss with no fuss at all (unlike setting-up a webserver). I need to read more into docker containers. Thanks for the suggestion @Alkarex
